src/oscar/apps/checkout/mixins.py

```diff
@@ -292,7 +292,7 @@ def send_confirmation_message(self, order, code, **kwargs):
             logger.warning("Order #%s - no %s communication event type",
                            order.number, code)
 
-    def get_message_context(self, order):
+    def get_message_context(self, order, code):
```

Comment:
I think `code` needs to default to `None` so as not to break backward compatibility with projects that have overridden this method and are not expecting a `code` argument.